WordPress/Classes/ViewRelated/System/FancyAlerts+AztecAnnouncement.swift

```diff
@@ -51,11 +51,10 @@ extension FancyAlertViewController {
         }
 
         let addConfetti: (FancyAlertButtonHandler) = { controller in
-            guard let imageView = controller.headerImageView else { return }
+            guard let imageView = controller.headerImageView, let imageViewHolder = imageView.superview else { return }
 
             let confettiView = ConfettiView.aztecAnnouncementConfettiView()
-            confettiView.frame = imageView.bounds.insetBy(dx: Constants.confettiViewInset,
-                                                          dy: Constants.confettiViewInset)
+            confettiView.frame = imageViewHolder.bounds.offsetBy(dx: 0, dy: Constants.confettiViewOffset)
```

Comment:
Hmm, they should be similar but you could be right… I think a -20px inset from the `imageViewHolder` would be the exact same.

To explain: this change is required because previously the image was always the width of the alert, minus the 40px of margin. Now the image will always be its actual size, centered, or scaled down proportionally if it can't fit. This was done so that the relatively _huge_ site address image scales down on the SE.